baybe/surrogates/base.py

```diff
@@ -124,60 +134,80 @@ def _make_target_scaler() -> OutcomeTransform | None:
 
     def _make_input_scaler(self, searchspace: SearchSpace) -> ColumnTransformer:
         """Make the input scaler for transforming computational dataframes."""
```

Comment:
What is a `computational dataframe`? Do you mean `dataframes containing parameters in computational representation`?

Reply:
Yes, exactly, only that the longer version would not fit into the single header line of the docstring. So how to proceed here?

Reply:
Same as [here](https://github.com/emdgroup/baybe/pull/325#discussion_r1698693855)